scraper.rb

```diff
@@ -22,7 +24,19 @@ def noko_for(url)
   Nokogiri::HTML(open(url).read)
 end
 
+# The terms of senators are 6 years long, but offset by two or four
+# years from their colleagues, so we're assuming that each year is a
+# different term.
+terms = (2015..Date.today.year).map do |year|
+  {
+    id: year,
+    start_date: Date.new(year, 3, 1).to_s,
+    end_date: Date.new(year, 11, 30).to_s,
+  }
```

Comment:
These are essentially magic numbers, which I think need to be explained somewhere.